src/Illuminate/Notifications/Console/stubs/notifications.stub

```diff
@@ -13,7 +13,7 @@ class CreateNotificationsTable extends Migration
     public function up()
     {
         Schema::create('notifications', function (Blueprint $table) {
-            $table->string('id')->primary();
+            $table->char('id', 36)->primary();
             $table->string('type');
```

Comment:
@ryanwinchester that's will be even better because of Postgres & MsSql native types.